src/OpenTelemetry.Exporter.Jaeger/JaegerTraceExporter.cs

```diff
@@ -15,16 +15,18 @@
 // </copyright>
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using OpenTelemetry.Exporter.Jaeger.Implementation;
 using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
 using OpenTelemetry.Trace.Export;
 
 namespace OpenTelemetry.Exporter.Jaeger
 {
-    public class JaegerTraceExporter : SpanExporter, IDisposable
+    public class JaegerTraceExporter : ActivityExporter, IDisposable
```

Comment:
This class is not required. We could remove this one. This is already a part of `JagerExporter.cs`

Reply:
@rajkumar-rangaraj , what about the method `ApplyLibraryResource`, that one is used in other places and we don't have in JaegerExporter

Reply:
Below one example:
```csharp
jaegerTraceExporter.ApplyLibraryResource(new Resource(new Dictionary<string, object>
            {
                ["Tag2"] = "value2",
            }));
```

should i remove those parts as well?

Reply:
we need just one class called JaegerExporter or JaegerTraceExporter containing all the export logic and implementing ActivityExporter

Reply:
ok. then i will pull that method to jaegerExporter